CHARTER.md

```diff
@@ -1,51 +1,61 @@
 
-Any voting period must last at least 7 days to account for different time zones and other obligations, and should last no longer than 30 days. The voting period should be stated in the proposal.
+A rejected proposal may only be submitted to the ART again after a miniumum of 30 days following a rejection. If the same proposal is rejected twice, it will not be considered again by the ART.
 
-ART members cast their votes by starting a review that either approves or rejects the decision suggested via a pull request. Once the voting period is over or all ART members have voted, the chairs record the decision, objections, and other relevant information in the `art-decisions` folder. Silence equals abstaining from a vote.
+ART members cast their votes by reviewing the pull request. An approval vote is cast by approving the pull request, and a rejection vote is cast by using the *Request changes* function and providing a reason for their rejection in the comment. Once the voting period is over or all ART members have voted, the chairs record the decision, objections, and other relevant information in the [`art-decisions` folder](art-decisions).
 
 ## Chair Elections
 
-If a chair leaves through non-participation or through their own announcement, elections for a new chair take place.
+If a chair leaves through non-participation, through their own announcement, or through the representative announcement of another chair, elections for a new chair take place.
+
+After the announcement of a chair leaving, there is a period of 30 days for nominations (from the date of the announcement). Any ART member can nominate another ART member or themselves, while a second member has to sponsor the nomination. Only members that have been part of the ART for at least six months are eligible to become a chair or sponsor a nomination.
+
+After the nomination period, elections take place asynchronously through the same process as standard proposals.
 
-After the announcement of a chair leaving, there is a period of 30 days for nominations (from the date of the announcement). Any ART member can nominate another ART member or themselves, while a second member has to sponsor the nomination. Only members that have been part of the ART for 6+ months are eligible to become a chair or sponsor a nomination.
+## Code of Conduct Enforcement
 
-After the nomination period, elections take place asynchronously through the same process as standard decision proposals.
+Removing or disciplining an ART member due to a violation of the [Arcalot Code of Conduct](https://github.com/arcalot/.github/blob/main/CODE_OF_CONDUCT.md) is performed by the chairs as a pull request to the ART. Only the the chairs vote on these matters, with an approval by 75% of the active chairs required for the matter to pass.
 
```

Comment:
I wonder if this should follow a process analogous to the joining process, in particular wrt to my suggestions above:  the process should essentially be made by unanimous consent of the chairs, with any dissenting vote triggering consideration via the normal proposal process.

Also, should there be an appeals process?  As things stand, if the community disagrees with the chairs' decision to remove a member, its only recourse is to remove and replace the approving chairs before reinstating the member.  It seems like, instead, there should be a way for the community to reconsider and override the decision by the chairs without necessitating a full revolt.

(Although, now that I look at it, I see where we describe how to elect a chair, but I don't see a process for removing one....)

Reply:
I think I'm OK with the idea of unanimous consent and an appeals process (beyond the pain of writing it and having more language to review).

I think the implied way of removing a chair is through the normal proposal process, which I think is actually good since it gives the broader community fair right to remove a chair without that chair having undue protection via a separate process.

Reply:
Given that we're not using unanimous consent for joining, I'm less enthused about using it for removals (see my comment below).

I support your notion that we should be able to remove a chair through the normal proposal process, but I don't think that that ability is actually implied by the document (although, I concur that there is probably nothing there which forbids it, either).

So, I recommend that we add a sentence, perhaps to the end of the _Chair Elections_ section indicating that chairs can be "recalled" via the normal proposal process.